package-lock.json

```diff
@@ -2665,6 +2665,66 @@
         "tslint": "^5.20.1",
         "webpack": "^4.0.0",
         "yorkie": "^2.0.0"
+      },
+      "dependencies": {
```

Comment:
I think this is what happens if you don't use the correct node/npm version. Try again with:
```
$ node --version
v14.15.0
$ npm --version
6.14.9
```

Reply:
kk, will do

Reply:
Updated!